msm/src/mvlookup.rs

```diff
@@ -328,40 +523,79 @@ pub mod prover {
+                .into_iter()
+                .map(|(id, evals)| {
+                    (
+                        id,
+                        Evaluations::<G::ScalarField, D<G::ScalarField>>::from_vec_and_domain(
+                            evals, domain.d1,
+                        ),
+                    )
+                })
+                .collect();
+
             let lookup_terms_poly_d1: Vec<DensePolynomial<G::ScalarField>> =
                 (&lookup_terms_evals_d1)
                     .into_par_iter()
                     .map(|lte| lte.interpolate_by_ref())
                     .collect::<Vec<_>>();
 
+            let fixed_lookup_tables_poly_d1: BTreeMap<ID, DensePolynomial<G::ScalarField>> =
+                (&fixed_lookup_tables_evals_d1)
+                    .into_par_iter()
+                    .map(|(id, evals)| (*id, evals.interpolate_by_ref()))
+                    .collect();
+
             let lookup_terms_evals_d8: Vec<Evaluations<G::ScalarField, D<G::ScalarField>>> =
                 (&lookup_terms_poly_d1)
                     .into_par_iter()
                     .map(|lte| lte.evaluate_over_domain_by_ref(domain.d8))
                     .collect::<Vec<_>>();
 
+            let fixed_lookup_tables_evals_d8: BTreeMap<
+                ID,
+                Evaluations<G::ScalarField, D<G::ScalarField>>,
+            > = (&fixed_lookup_tables_poly_d1)
+                .into_par_iter()
+                .map(|(id, poly)| (*id, poly.evaluate_over_domain_by_ref(domain.d8)))
+                .collect();
+
             let lookup_terms_comms_d1: Vec<PolyComm<G>> = (&lookup_terms_evals_d1)
                 .into_par_iter()
                 .map(|lte| srs.commit_evaluations_non_hiding(domain.d1, lte))
                 .collect::<Vec<_>>();
 
+            let fixed_lookup_tables_comms_d1: BTreeMap<ID, PolyComm<G>> =
+                (&fixed_lookup_tables_evals_d1)
+                    .into_par_iter()
+                    .map(|(id, evals)| (*id, srs.commit_evaluations_non_hiding(domain.d1, evals)))
+                    .collect();
+
             lookup_terms_comms_d1
                 .iter()
                 .for_each(|comm| absorb_commitment(fq_sponge, comm));
+
+            fixed_lookup_tables_comms_d1
+                .values()
+                .for_each(|comm| absorb_commitment(fq_sponge, comm));
             // -- end computing the row sums h
 
             // -- start computing the running sum in lookup_aggregation
-            // The running sum, \phi, is defined recursively over the subgroup as followed:
-            // - phi(1) = 0
-            // - phi(\omega^{j + 1}) = \phi(\omega^j) + \
-            //                         \sum_{i = 1}^{n} (1 / \beta + f_i(\omega^{j + 1})) - \
-            //                         (m(\omega^{j + 1}) / beta + t(\omega^{j + 1}))
-            // - phi(\omega^n) = 0
+            // The running sum, φ, is defined recursively over the subgroup as followed:
+            // - φ(1) = 0
```

Comment:
Nice comment. Comment for myself: for the trans-circuit MVLookup we'll have to set these 1/n values to previous and next accumulator.